scalac-scoverage-runtime/shared/src/main/scala/scoverage/Invoker.scala

```diff
@@ -67,6 +146,35 @@ object Invoker {
     override def accept(pathname: File): Boolean = pathname.getName.startsWith(MeasurementsPrefix)
   })
 
+
+  def resetDataDir(dataDir: String): Unit = {
+    // Making the directory if it does not exist
+    new File(dataDir).mkdirs()
+
+    // clean the directory
+    findMeasurementFiles(dataDir).foreach(_.delete)
+  }
+
+  /**
+   *  Copy coverage file from [src] dir to [dest] dir
+   * @param src directory path for source of coverage sfile
+   * @param dest directory path for destination of coverage file
+   */
+  def copyCoverageFile(src: String, dest: String) = {
+    Files.copy(
+      Paths.get(s"$src/$CoverageFileName"),
+      Paths.get(s"$dest/$CoverageFileName"),
+      StandardCopyOption.REPLACE_EXISTING
+    )
+  }
+
+  def safe_name(str: String): String =  {
+    val pattern = "[^/]+".r
```

Comment:
This method should have a camel-case name.

But also: you should watch out for turning filepaths into single names, there are filename length limits, and take a bunch of filenames and merging them into one is very likely to violate that.

Instead, you should consider preserving the full name of the file under the directory, and call `mkdirs` for the parents of each file.

Reply:
The method does not exists in the latest PR.